scripts/install_gradio.sh

```diff
@@ -7,3 +7,4 @@ pip_required
 
 echo "Installing Gradio..."
 pip install -e .
+pip install -e client/python
```

Comment:
I actually think the client should *not* be editable installed. Rather, it should pull the latest from pypi as this is consistent with how users will be using the gradio library. 

Suppose the following happens:
* We make some changes in both the client library and gradio library (e.g. add a util to the client library, and add some feature in the gradio library which uses this util) 
* If the client is editable installed, then all tests will continue to pass, and we won't realize that we've made a breaking change. We could release the next version of the gradio library thinking everything is working fine because all tests are passing. But, in reality, gradio won't work properly
* If the client is installed from pypi, then the gradio tests will fail because the tests will correctly catch that we have not made a gradio client release to pypi yet.